hack/make/vars.mk

```diff
@@ -37,11 +37,12 @@ BUNDLE_METADATA_OPTS ?= $(BUNDLE_CHANNELS) $(BUNDLE_DEFAULT_CHANNEL)
 # BUNDLE_IMG defines the image:tag used for the bundle.
 # You can use it as an arg. (E.g make bundle-build BUNDLE_IMG=<some-registry>/<project-name-bundle>:<tag>)
 BUNDLE_IMG ?= $(IMAGE_REGISTRY)/$(REGISTRY_NAMESPACE)/$(BUNDLE_IMAGE_NAME):$(IMAGE_TAG)
+RAMEN_BUNDLE_IMAGE ?= quay.io/ramendr/ramen-hub-operator-bundle:canary
```

Comment:
This is unrelated to dependencies and helps with testing.
Keep this in separate commit.